### PR TITLE
Improve code

### PR DESCRIPTION
**Pull request by RouNNdeL**

### CODE REVIEW

**Comment by MBWlodarczyk** [comment]
very improved gz

**Comment by RouNNdeL** [comment]
> very improved gz

dzieki, merging

File: metaparser/modules/openXml.py [hunk 1]
```diff
@@ -1,8 +1,8 @@
-import re  # type: ignore
-import shutil  # type: ignore
-import tempfile  # type: ignore
-import xml.etree.ElementTree  # type: ignore
-import zipfile  # type: ignore
+import re
```
Comment (MBWlodarczyk) [inline metaparser/modules/openXml.py:1]: wow type no longer ignored??? nice

Comment (RouNNdeL) [reply metaparser/modules/openXml.py:1]: daduez ignored, we not ignore

**Pull request merged**